antora-ui-camel/src/partials/header-content.hbs

```diff
@@ -23,6 +23,7 @@
         </div>
       </div>
       <div class="navbar-fill"></div>
+      <div class="break-row"></div>
```

Comment:
Doesn't seem to make a difference, I've removed this from the DOM in developer tools and resized from 1023px to 300px, and the layout remained the same.